pallets/manta-pay/src/lib.rs

```diff
@@ -680,20 +680,19 @@ pub mod pallet {
             checkpoint: Checkpoint,
             max_receivers: u64,
             max_senders: u64,
-        ) -> DensePullResponse {
+        ) -> RuntimeDensePullResponse {
             let (more_receivers, receivers) =
                 Self::pull_receivers(*checkpoint.receiver_index, max_receivers);
             let (more_senders, senders) = Self::pull_senders(checkpoint.sender_index, max_senders);
             let senders_receivers_total = (0..=255)
                 .map(|i| ShardTrees::<T>::get(i).current_path.leaf_index as u128)
                 .sum::<u128>()
                 + NullifierSetSize::<T>::get() as u128;
-            DensePullResponse {
+            RuntimeDensePullResponse {
                 should_continue: more_receivers || more_senders,
-                receivers: base64::encode(receivers.encode()),
-                senders: base64::encode(senders.encode()),
+                receivers: receivers.encode(),
```

Comment:
since you're dropping the base64 encoding here, is it still accurate to call it `Dense`  or should it just be `RuntimePullResponse`

Reply:
This is much better than before actually. The runtime (single threaded, responsible for building/verifying blocks) hands off the base64 encoding workload to the other CPU cores that run the client and RPC now, so it should be much more parallelizable without choking out the runtime.

Reply:
If rpc request is sent to full node, block production will not be affected from collator.

Reply:
Original question still stands tho

> you're dropping the base64 encoding here, is it still accurate to call it Dense or should it just be RuntimePullResponse

Reply:
I have removed `RuntimeDensePullResponse` and `pull_ledger_diff`, so it might not confuse you anymore.